### PR TITLE
feat(appbar): added appbar

### PR DESCRIPTION
**Pull request by yishain11**

### CODE REVIEW

**Comment by yishain11** [comment]
PR for appbar.
 log for sowhing state fn is just because the ts error when not using it.
the usage is a part of advance use, need to talk it over during review.

File: frontend/src/components/Appbar/types.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export interface AppbarProps {
+  agreementName: string;
+  closeFn: () => void;
+}
```
Comment (avirankatz) [inline frontend/src/components/Appbar/types.ts:4]: Let's move this into `Appbar.tsx` and remove Appbar dir (same thing we did with other components)

File: frontend/src/components/Sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -58,7 +58,7 @@ const Sidebar: FC<IFCProps> = ({ mobileOpen, handleSidebarToggle }) => {
   const content = (
     <>
       <SC.LogoContainer>
-        <Link to="/" title={t(StringBank.GOTO_HOMEPAGE_TITLE)}>
+        <Link to="/" title={t(StringBank.GOTO_HOMEPAGE_TITLE)} style={{ display: 'flex' }}>
```
Comment (avirankatz) [inline frontend/src/components/Sidebar/Sidebar.tsx:61]: Is this intended?

File: frontend/src/pages/NewAgreement.tsx [hunk 3]
```diff
@@ -74,42 +75,50 @@ const NewAgreement: FC = () => {
...
-          >
-            {step === 3 ? t(StringBank.PUBLISH_AGREEMENT) : t(StringBank.CONTINUE)}
-          </Button>
-          {step == 3 && (
-            <Button variant="text" onClick={() => setStep(step - 1)}>
-              {t(StringBank.BACK)}
-            </Button>
+    <>
+      <Appbar
+        agreementName={agreementName || 'My New Agreement'}
```
Comment (avirankatz) [inline frontend/src/pages/NewAgreement.tsx:80]:
```suggestion
        agreementName={agreementName}
```

File: frontend/src/pages/NewAgreement.tsx [hunk 4]
```diff
@@ -74,42 +75,50 @@ const NewAgreement: FC = () => {
...
+        closeFn={() => {
+          console.log('closing appbar');
```
Comment (avirankatz) [inline frontend/src/pages/NewAgreement.tsx:82]: Let's replace this with navigation back. We may use `useNavigation` from react router

File: frontend/src/components/Appbar/Appbar.tsx [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+  }
+`;
+
+const Appbar: FC<AppbarProps> = (props) => {
+  const steps = ['Rationale', 'Sections', 'Rules'];
+  const [activeStep, setActiveStep] = useState(0);
+  const [completed, setCompleted] = useState<{
+    [k: number]: boolean;
+  }>({});
+  console.log('setCompleted', setCompleted);
```
Comment (avirankatz) [inline frontend/src/components/Appbar/Appbar.tsx:35]: Could you please remove this print?

**Review by avirankatz — CHANGES_REQUESTED** [review]
Please see requested changes

**Review by avirankatz — APPROVED** [review]

**Pull request merged**